samples/sensor/lsm6dsl/sample.yaml

```diff
@@ -5,3 +5,9 @@ tests:
     harness: console
     tags: sensors
     depends_on: lsm6dsl
+    harness_config:
+      type: multi_line
```

Comment:
Why not using the 'ordered' rule?
ordered: true

Maybe it does not add a big added value, but I would use it.
What do you think?

Reply:
Looking into it. So far, I don't manage to get test passing. Seems like regex not good enough.